modules/local/openms_maprttransformer.nf

```diff
@@ -18,7 +18,7 @@ process OPENMS_MAPRTTRANSFORMER {
         task.ext.when == null || task.ext.when
 
     script:
-        def prefix           = task.ext.prefix ?: "${meta.id}_aligned"
+        def prefix           = task.ext.prefix ?: "${alignment_file.baseName}_aligned"
```

Comment:
Does this differ from meta?

Reply:
No, actually it could be {meta.id + meta.sample + meta.condition} I think. Meta.id I found to less elaborate when debugging..

Reply:
I guess that is due to the new changes that you made

Reply:
I'll check again